sklearn/cross_validation.py

```diff
@@ -33,14 +33,12 @@
 from .gaussian_process.kernels import Kernel as GPKernel
 from .exceptions import FitFailedWarning
 
-
```

Comment:
Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.

Reply:
Sorry! My IDE made those changes while formatting, I'll revert them